tsec-http4s/src/main/scala/tsec/authentication/EncryptedCookieAuthenticator.scala

```diff
@@ -454,20 +448,17 @@ object EncryptedCookieAuthenticator {
           }
         }
 
-      def refresh(authenticator: AuthEncryptedCookie[A, I]): OptionT[F, AuthEncryptedCookie[A, I]] =
+      def refresh(authenticator: AuthEncryptedCookie[A, I]): F[AuthEncryptedCookie[A, I]] =
         settings.maxIdle match {
           case Some(_) =>
-            OptionT
-              .liftF(
-                F.delay(
-                  authenticator.copy[A, I](
-                    lastTouched = Some(Instant.now())
-                  )
+            F.delay(
```

Comment:
For consistency `F.delay(Instant.now()).flatMap( .. )`

Reply:
Wait why? We're delaying the evaluation of `Instant.now` anyway, but we avoid one flatMap op. I think this is a small micro optimization.